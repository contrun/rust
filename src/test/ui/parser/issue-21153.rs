trait MyTrait<T>: Iterator {
    Item = T;
    //~^ ERROR non-item in item list
}

fn main() {}
