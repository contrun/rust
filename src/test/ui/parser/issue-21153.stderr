error: non-item in item list
  --> $DIR/issue-21153.rs:2:5
   |
LL | trait MyTrait<T>: Iterator {
   |                            - item list starts here
LL |     Item = T;
   |     ^^^^ non-item starts here
LL |
LL | }
   | - item list ends here

error: aborting due to previous error

